src/test/java/seedu/taassist/logic/commands/CommandTestUtil.java

```diff
@@ -36,6 +39,9 @@ public class CommandTestUtil {
     public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
     public static final String VALID_CLASS_CS1101S = "CS1101S";
     public static final String VALID_CLASS_CS1231S = "CS1231S";
+    public static final String VALID_SESSION_LAB1 = "LAB1";
+    public static final String VALID_DATE = "2022-01-01";
+    public static final Double VALID_GRADE_VALUE_12345 = 12.345;
```

Comment:
Should this be changed to `String`? Since all the other constants here are declared as `String` and parsed inside the test cases when needed. For example the `VALID_DATE` constant.